_security/access-control/rest-layer-authz.md

```diff
@@ -16,6 +16,8 @@ Developers, on the other hand, will need to understand the ideas behind `NamedRo
 
 The benefits of using the REST layer for authorization include the ability to authorize requests at the REST layer and filter out unauthorized requests. As a result, this decreases the processing burden on the transport layer while allowing granular control over access to APIs.
 
+Some read operations manage state (for example [scroll]({{site.url}}{{site.baseurl}}/api-reference/scroll/). It is, therefore, recommended to control read and write access using Security plugin [permissions]({{site.url}}{{site.baseurl}}/security/access-control/permissions/), instead of allowing/blocking HTTP request verbs.
```

Comment:
```suggestion
Some read operations, such as  [scroll]({{site.url}}{{site.baseurl}}/api-reference/scroll/), manage state. Therefore, it is recommended to control read and write access using the Security plugin [permissions]({{site.url}}{{site.baseurl}}/security/access-control/permissions/), instead of allowing/blocking HTTP request verbs.
```